openedx/core/djangoapps/content_libraries/constants.py

```diff
@@ -0,0 +1,19 @@
+""" Constants used for the content libraries. """
+
+from django.utils.translation import ugettext_lazy as _
+
+# This API is only used in Studio, so we always work with this draft of any
```

Comment:
Nit: by moving this constant into `constants.py` instead of `api.py`, it's not quite as clear what "This API" refers to. The code in `api.py` and `views.py` is only used in Studio, but the code in `library_context.py` which implements the "Learning Context API" is used in the LMS and Studio.